lwt/tls_lwt.ml

```diff
@@ -141,7 +140,16 @@ module Unix = struct
           | `Eof     -> fail End_of_file
           | `Ok cs   -> push_linger t cs ; drain_handshake t
 
-  let reneg ?authenticator ?acceptable_cas ?cert ?(drop = true) t =
+  let default_timeout = Duration.of_sec 60
+
+  let with_timeout timeout f =
+    if timeout > 0L then
```

Comment:
I'm a bit sad about making a time span of zero a magic value that disables timeouts. Other time libraries I've worked with generally provide something like [Time.max_value] if you really want to never time out.

I don't think I'd want to implement a similar check in tls_async, and would weakly prefer it went away here to keep the behavior relatively consistent across concurrency abstractions.

Reply:
ok, we can as well use `Int64.max_int` and no conditional in `with_timeout`. Thanks for the suggestion.

Reply:
done in 507fade